spec/features/students/visit_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+feature "visit students index", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.